python/src/cloudstorage/cloudstorage_api.py

```diff
@@ -273,6 +277,184 @@ def listbucket(path_prefix, marker=None, prefix=None, max_keys=None,
   return _Bucket(api, bucket, options)
 
 
+# pylint: disable=too-many-locals, too-many-branches, too-many-statements
+def compose(list_of_files, destination_file, files_metadata=None,
+            content_type=None, retry_params=None, _account_id=None):
+  """
+    Runs the GCS Compose on the given files.
```

Comment:
See https://www.python.org/dev/peps/pep-0257/
You should use one of these styles:
1) """Runs the GCS ...
2) """
    Runs the GCS

Though I like the latter the former seems to be consistent with existing docstring in this file.